content/en/aws/opensearch/index.md

```diff
@@ -0,0 +1,415 @@
+---
+title: "OpenSearch Service"
+linkTitle: "OpenSearch Service"
+categories: ["LocalStack Community"]
+description: >
+  Amazon OpenSearch Service
+---
+
+The OpenSearch Service in LocalStack lets you create one or more single-node OpenSearch clusters that behave like the [Amazon OpenSearch Service](https://aws.amazon.com/opensearch-service/).
+
+
+## Creating an OpenSearch cluster
+
+You can go ahead and use [awslocal]({{< ref "aws-cli.md#localstack-aws-cli-awslocal" >}}) to create a new opensearch domain via the `aws opensearch create-domain` command.
```

Comment:
```suggestion
You can go ahead and use [awslocal]({{< ref "aws-cli.md#localstack-aws-cli-awslocal" >}}) to create a new OpenSearch domain via the `aws opensearch create-domain` command.
```
or is there a reason why this is the sole occurrence where OpenSearch is not correctly capitalized?

Reply:
Thanks, fixed!